apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/kontoregisterservice/KontoregisterClient.java

```diff
@@ -57,6 +55,8 @@ public void gjenopprett(RsDollyUtvidetBestilling bestilling, DollyPerson dollyPe
 
     @Override
     public void release(List<String> identer) {
-        // Kontoregister har ikke sletting
+        identer.stream().forEach(ident -> kontoregisterConsumer.sendSlettKontoRequest(ident)
+                .doOnSuccess(c -> log.info("slettet konto for " + ident))
+                .block());
```

Comment:
Ikke bruk .block() her, da blir denne hengende og vente til ferdig. Her logges det også per ident, i stedet for en gang for hele operasjonen.
Se på en av de andre og benytt pattern med Flux .subscribe slik at dette kjører asynkront.

Reply:
fikset det